projects/lib/src/board/westernboard.cpp

```diff
@@ -94,6 +95,13 @@ void WesternBoard::vInitialize()
 	m_rookOffsets[1] = -1;
 	m_rookOffsets[2] = 1;
 	m_rookOffsets[3] = m_arwidth;
+
+	m_pawnAmbiguous = pawnAmbiguity(StepType::Free);
+}
+
+inline int WesternBoard::getPawnPush(const PawnStep& ps, int sign) const
```

Comment:
I do not know actually. I hope this is not a rhetoric question. I will investigate this. If I am not sure it works I will move the function body to the header file.